app/scripts/modules/core/src/managed/NoticeCard.tsx

```diff
@@ -6,15 +6,16 @@ import { Icon, IconNames } from '../presentation';
 import styles from './NoticeCard.module.css';
 
 interface INoticeCardProps {
-  title: JSX.Element | string;
-  text: JSX.Element | string;
+  title: React.ReactNode;
+  actions?: React.ReactNode;
+  text?: React.ReactNode;
```

Comment:
in an exciting twist, turns out `JSX.Element` expects _exactly one_ element (and it has to be an actual element), whereas `React.ReactNode` supports any of the things you can render using `{thingThatIsRenderable}` — including strings. TIL.